frontend/src/components/course-component.tsx

```
@@ -14,6 +20,16 @@ const CourseComponent = ({ name, credits, grade }: CourseProps) => {
   const { setCourseToModify } = useCourseToModify();
   const { courses, deleteCourse } = useCourseStore();
 
+	const sanitizeInput = (input: string): string => {
+		return input.replace(/<script>/gi, '').replace(/javascript:/gi, '');
+	};
+
+	const renderUnsafeContent = (content: string) => {
+		return (
+			<div dangerouslySetInnerHTML={{ __html: content }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)